### PR TITLE
Create new_issues_to_protocol_design_project

### PR DESCRIPTION
**Pull request by gordsport**
A Github action to move any new issues raised against this repo to the Protocol Design project board to have a single clear view of work required on protocol design.

### CODE REVIEW

File: .github/workflows/new_issues_to_protocol_design_project [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Add new issues to org Protocol Design project
+    steps:
+      - uses: alex-page/github-project-automation-plus@v0.7.1
```
Comment (fkondej) [inline .github/workflows/new_issues_to_protocol_design_project]: Let's follow good practices for [Using third-party actions](https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions), and `Pin actions to a full length commit SHA`.
Currently the latest version is [v0.7.1](https://github.com/alex-page/github-project-automation-plus/releases/tag/v0.7.1) which corresponds to `50502d399cbb98cefe7ce1f99f93f78c6756562e`.
So please change:
```suggestion
      - uses: alex-page/github-project-automation-plus@50502d399cbb98cefe7ce1f99f93f78c6756562e
```

File: .github/workflows/new_issues_to_protocol_design_project [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        with:
+          project: Protocol Design
+          column: To do
+          repo-token: ${{ secrets.GH_NEW_CARD_TO_PROJECT }}
```
Comment (fkondej) [inline .github/workflows/new_issues_to_protocol_design_project]: Please add an empty line at the end.

Comment (fkondej) [inline .github/workflows/new_issues_to_protocol_design_project]: Is this GitHub Access Token (`GH_NEW_CARD_TO_PROJECT` secret value) for your account?

**Review by fkondej — CHANGES_REQUESTED** [review]
Minor changes and question.

**Review by pennyandrews — COMMENTED** [review]
Looks good to me, so if Fryd is happy then all good with me.

File: .github/workflows/new_issues_to_protocol_design_project (hunk 2 above)
Comment (gordsport) [reply .github/workflows/new_issues_to_protocol_design_project]: Yes this is a GitHub Access Token I created against my account - with ‘repo’ and ‘write:org’ access as tested in the repoplayground repo. For actions like these should we be using a different account for the access token?
The token was then applied as an org secret as the intention was this action would be repeated across a few repos thus could use a common token.

Comment (fkondej) [reply .github/workflows/new_issues_to_protocol_design_project]: Secret value got updated to use bot's access token. You can find it in 1password.

**Review by fkondej — APPROVED** [review]

**Pull request merged**